tools/build/Makefile

```diff
@@ -41,6 +41,8 @@ $(PYTHON_DIR) : $(DSTDIR)
 	. $(abspath $(DSTDIR)/bin/activate) ; pip install --upgrade colorlog
 	. $(abspath $(DSTDIR)/bin/activate) ; pip install --upgrade twisted
 	. $(abspath $(DSTDIR)/bin/activate) ; pip install --upgrade protobuf
+	. $(abspath $(DSTDIR)/bin/activate) ; apt-get update
+	. $(abspath $(DSTDIR)/bin/activate) ; apt-get install -y python3-tk
```

Comment:
python3-tk can be installed in docker/Dockerfile.tcf-dev

Reply:
Done.